### DISCUSSION TITLE
Meeting Page MISSING (?) Upgraded to 3.11.0

### DISCUSSION THREAD

**Discussion by lloy0076** [code4recovery/12-step-meeting-list#431] [Q&A] [answered]
**Describe the bug**
I upgraded from 3.10.0 to 3.11.0 and the meeting page is "just gone".

**To Reproduce**
1. Install WordPress
2. Install 3.10.0
3. Upgrade

**Expected behavior**
https://aa-stlawrenceny.org/meetings-by-day/?tsml-day=any should show several meetings.

**Screenshots**
![image](https://user-images.githubusercontent.com/1174532/125163471-5177a080-e15b-11eb-9f24-88a5b36cd27a.png)

**Additional context**
![image](https://user-images.githubusercontent.com/1174532/125163492-5f2d2600-e15b-11eb-8512-e7dcefb4a0cb.png)

[meeting-schedule(16).pdf](https://github.com/code4recovery/12-step-meeting-list/files/6795288/meeting-schedule.16.pdf)

The system KNOWS the data is there.

It's possible the page for displaying the list of meetings have changed.

**Comment by lloy0076** [comment]
It does seem that the short codes do still work, so `[tsml_group_count]` shows the right number of groups:

![image](https://user-images.githubusercontent.com/1174532/125164096-7ae5fb80-e15e-11eb-9726-f9da7ba5ad68.png)

**Comment by tech2serve** [comment]
Converting to our new Discussions page.

**Comment by tech2serve** [comment, marked as answer]
I am able to navigate to your meeting page using [this link](https://aa-stlawrenceny.org/?post_type=tsml_meeting).

Try navigating to Settings --> Permalinks, and then click on Save to refresh permalinks.

**Reply by lloy0076** [reply]
Well, that fixed it - although I cannot figure out why my site would have needed to refresh its permalinks.

**Reply by lloy0076** [reply]
Actually did the change from `meetings_by_day` to `meetings` do that?

**Reply by tech2serve** [reply]
Frankly, I'm not sure. But I do believe that modifications made to the permalinks (custom links) can result in the need to refresh all permalinks. 

Luckily, it is an easy fix. :-)
